lib/src/rows/Rows.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import styled from "styled-components";
+import { RowProps, RowsProps } from "./types";
+
+const DxcRows = ({
+  alignX = "stretch",
+  alignY = "stretch",
+  as = "div",
+  divider = false,
+  gutter = "0rem",
+  reverse = false,
+  children,
+}: RowsProps): JSX.Element => {
+  return (
+    <Rows as={as} alignX={alignX} gutter={gutter} reverse={reverse}>
+      {React.Children.map(children, (child, index) => {
+        return (
+          <>
+            {child.type === DxcRows.Row ? (
+              React.cloneElement(child, { alignY })
+            ) : (
+              <DxcRow alignY={alignY}>{child}</DxcRow>
+            )}
+            {divider && index !== React.Children.count(children) - 1 && <Divider />}
+          </>
+        );
+      })}
+    </Rows>
+  );
+};
+
+const DxcRow = ({ height = "auto", alignY, children }: RowProps): JSX.Element => {
+  return (
+    <Row height={height} alignY={alignY}>
+      {children}
+    </Row>
+  );
+};
+
+DxcRows.Row = DxcRow;
+
+const Rows = styled.div<RowsProps>`
+  display: flex;
+  ${({ alignX, gutter, reverse }) => `
+    flex-direction: ${reverse ? "column-reverse" : "column"};
+    align-items: ${alignX};
```

Comment:
I didn't think about this the first time, but, why are we using `align-items` (which aligns items on the cross axis) to align items on the main axis? Shouldn't we be using `justify-content` for that purpose?

![image](https://user-images.githubusercontent.com/44321109/175944659-8412a27b-db4f-43f8-96f7-a4e4f7b62d92.png)

from: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Aligning_Items_in_a_Flex_Container

Reply:
I know it's working but I find it a bit confusing...

Reply:
We are using a flex with column orientation, so I think this changes the order of the axes: `align-items` for the main axis and `justify-content` for the cross axis.

Reply:
You are right!